pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -88,6 +91,23 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 
 	tc.SetDefaults(ctx)
 
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: tc.GetSpec().GetTargetNamespace(),
+			Labels: map[string]string{
+				"operator.tekton.dev/targetNamespace": tc.GetSpec().GetTargetNamespace(),
+			},
+		},
+	}
+	_, err := r.kubeClientSet.CoreV1().Namespaces().Get(ctx, tc.GetSpec().GetTargetNamespace(), metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			if _, err = r.kubeClientSet.CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{}); err != nil {
+				return err
+			}
+		}
+	}
+
```

Comment:
```suggestion
	r.createTargetNamespace()
```

please move this into a function.

There are 2 ways how "functions" help us in coding:

1. the obvious, code reuse.
2. greatly improved readability. ie, when some one reads the ReconcileKind() what need to be seen is something like `ensureTargetNamespace()`. Keeping the full implementation here is distracting.

Whenever we write code, if a sequence of code lines explain more than the intention of the current function, we are going too low level. That is, the implementaiton of a sub mechanisms could be written as functions even if they might not be reused else where.